content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,300 @@
+                ...
+
+$ dvc exp run -S myparams.toml:learning_rate = 0.0001
+```
+
+</details>
+
+### Updating experiment parameters on-the-fly
+
+DVC allows to update the parameters from command line when running
+`dvc experiments`. The `--set-param` (`-S`) option takes a parameter name and
+its value, and updates the params file before the run.
+
+```dvc
+$ dvc exp run --set-param model.learning_rate=0.0002
+```
+
+> Note that parameters are attached to experiments so you can view them together
+> with `dvc exp show` and `dvc exp diff`.
+
+To set more than one param for the same experiment, use the `-S` option multiple
+times, or supply a comma-delimited list to `-S`:
+
+```dvc
+$ dvc exp run -S learning_rate=0.001 -S units=128
+# is equivalent to
+$ dvc exp run -S learning_rate=0.001,units=128
+```
+
+> ⚠️ Note that DVC doesn't check whether parameters given to `--set-param` are
+> already in the parameters file. If there is a typo, a new or different param
+> will be added/changed.
+
+## The experiments queue
+
+The `--queue` option of `dvc exp run` tells DVC to append an experiment for
+later execution. Nothing is actually run yet.
+
+<details>
+
+### How are experiments queued?
+
+Queued experiments are created similar to
+[Git stash](https://www.git-scm.com/docs/git-stash). The last experiment queued
+is found in `.git/refs/exps`, and earlier ones are in its [reflog].
+
+[reflog]:
+  https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefreflogareflog
+
+</details>
+
+```dvc
+$ dvc exp run --queue -S units=10
+Queued experiment '1cac8ca' for future execution.
+$ dvc exp run --queue -S units=64
+Queued experiment '23660bb' for future execution.
+$ dvc exp run --queue -S units=128
+Queued experiment '3591a5c' for future execution.
+$ dvc exp run --queue -S units=256
+Queued experiment '4109ead' for future execution.
+```
+
+> `dvc exp show` will mark queued experiments with an asterisk `*`.
+
+Each experiment is derived from the workspace at the time it's queued. If you
+make changes in the workspace afterwards, they won't be reflected in queued
+experiments (once run).
+
+<details>
+
+### How are queued experiments isolated? (Temporary directories)
+
+To guarantee that queued experiments derive from their original workspace, DVC
+creates a copy of it in `.dvc/tmp/exps/`, where the experiment will run. All
+these workspaces share the main project <abbr>cache</abbr>.
+
+If you want to isolate an experiments this way without queuing it, you can use
+the `--temp` option. This allows you to continue working while a long experiment
+runs.
+
+```dvc
+$ nohup dvc exp run --temp &
+[1] 30473
+nohup: ignoring input and appending output to 'nohup.out'
+```
+
+> The above example creates a `nohup.log` file in the original workspace with
+> the output of the DVC process.
+
+Note that Git-ignored files/dirs are explicitly excluded from queued/temp runs
+to avoid committing unwanted files into final ([persisted]) experiments.
```

Comment:
This is minor, but I don't think this should be strictly about final/persisted experiments. For example, someone can `dvc exp push` and expose sensitive info.

Reply:
Good catch. I rephrased a little in 85a38c97.